.github/workflows/test.yaml

```diff
@@ -55,7 +55,6 @@ jobs:
           php-version: ${{ matrix.php-version }}
           coverage: pcov
           tools: composer:v2
-
```

Comment:
this space removal looks wrong

Reply:
I didn't touch the file, so it's certainly done by some dev-kit changes.